test/controllers/api/v2/organizations_controller_test.rb

```diff
@@ -81,4 +81,14 @@ class Api::V2::OrganizationsControllerTest < ActionController::TestCase
     assert_response :success
     assert_equal 1, organization.reload.organization_parameters.count
   end
+
+  test "should not update invalid organization" do
+    put :update, params: { :id => Organization.first.to_param, :organization => { :name => "" } }
```

Comment:
Hey, the `to_param` method returns `/organiazation/1` while here you want to put just number instead. Try `Organization.first.id` instead.